### DISCUSSION TITLE
Clustering mode connection

### DISCUSSION THREAD

**Discussion by garry-t** [ruby-amqp/bunny#683] [General]
In docs absent if possible or no to set multiple cluster node for connection with `automatically_recover=true`

```ruby
conn = Bunny.new(:host => [{"localhost"}, {"localhost"}, {"localhost"}], :vhost => "myapp.production", :user => "bunny", :password => "t0ps3kret")
```
Task: I have 7 cluster nodes. I assume that client able to consume as many node as needed and switch between them in case connection issue. 
Or bunny is not support cluster mode ?

**Comment by michaelklishin** [comment]
You [assumption is wrong](https://www.rabbitmq.com/docs/clustering#clustering-and-clients), a client only connects to one node at a time.

Bunny does support [endpoint lists](https://github.com/ruby-amqp/bunny/blob/main/spec/higher_level_api/integration/connection_recovery_spec.rb#L449) but 

```
[{"localhost"}, {"localhost"}, {"localhost"}]
```

is not valid Ruby code.

RabbitMQ Stream protocol clients [can connect to multiple nodes in paraller](https://www.rabbitmq.com/blog/2021/07/23/connecting-to-streams) but Bunny is not one of them.
